### PR TITLE
Change to instance access

### PR DESCRIPTION
**Pull request by blade254353074**

### CODE REVIEW

**Comment by shellscape** [comment]
What problem does this solve? On the surface it looks like this is bikeshedding.

File: dist/index.js [hunk 1]
```diff
@@ -20,6 +20,44 @@ var _promise = require('babel-runtime/core-js/promise');
 
 var _promise2 = _interopRequireDefault(_promise);
 
+exports.default = function (options) {
```
Comment (shellscape) [inline dist/index.js]: please don't submit changes to `dist`

Comment (blade254353074) [inline dist/index.js]: Sorry for that.

**Comment by blade254353074** [comment]
I think we need to expose the configured middleware instance rather than not configured middleware method.

**Comment by shellscape** [comment]
That's reasonable and a good catch. Unfortunately I feel that too much was done with this PR along personal code style and naming preferences to achieve that. I'll treat this as a feature change request and will publish a new minor version with the change, since it'll be a breaking, corrective change.

Thanks again for taking the time to open a PR.

**Comment by blade254353074** [comment]
All right, waiting for your good news~

**Pull request closed without merging**